mvc/src/main/java/com/interface21/webmvc/servlet/mvc/DispatcherServlet.java

```diff
@@ -65,11 +59,17 @@ protected void service(final HttpServletRequest request, final HttpServletRespon
             final var handlerAdapter = handlerAdapterRegistry.getHandlerAdapter(handler);
             final ModelAndView modelAndView = handlerAdapter.handle(request, response, handler);
 
-            final View view = modelAndView.getView();
-            view.render(new HashMap<>(), request, response);
+            render(request, response, modelAndView);
```

Comment:
이렇게 되니 해당 로직을 따로 분리하지 않아도 될 것 같네요 ~ (반영할 필요 없어요~)

```suggestion
            View view = modelAndView.getView();
            view.render(modelAndView.getModel(), request, response);
```

Reply:
처음에 단순하게 작성했다가

1. `HandlerAdapter`를 통해 `ModelAndView`를 얻는다.
2. `ModelAndView`에 있는 `Model`과 `View`로 render를 수행한다.

이런 식으로 로직의 흐름을 가져가고, 보는 사람도 그렇게 느끼게 하고 싶어서, 따로 메서드로 분리했습니다.

그렇지만, view를 render 한다는 관점에서는 view에 model을 전달하는 게 더 맞을 수도 있겠다는 생각도 듭니다. 실제로 사용하는 메서드도 `view.render()`니까요. 그래서 켬미가 추천해준 방식도 맞다고 생각합니다.